cypress/integration/plugins/reports-dashboards/04-download.spec.js

```diff
@@ -91,9 +73,7 @@ describe('Cypress', () => {
     cy.wait(5000);
 
     // open saved search list
-    cy.get(
-      'button.euiButtonEmpty:nth-child(3) > span:nth-child(1) > span:nth-child(1)'
-    ).click({ force: true });
+    cy.get('[data-test-subj="discoverOpenButton"]').click({ force: true });
```

Comment:
I assume the old one is not working any more because of the new Discover button

Reply:
![Screenshot 2023-09-19 at 10 38 41 AM](https://github.com/opensearch-project/opensearch-dashboards-functional-test/assets/26328171/cd703c17-2b0a-4b3e-8bb3-365b0801a913)

Reply:
Attached screenshot of this test passing with data-test-subj method